src/ReplicatedStorage/Client/InterfaceComponents/TextLabel.lua

```diff
@@ -0,0 +1,91 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local Packages = ReplicatedStorage:WaitForChild("Packages")
+local Fusion = require(Packages.Fusion)
+local WrappedFusion = require(Packages.FusionUtils).Wrapped
```

Comment:
What's FusionUtils.Wrapped?

Reply:
FusionUtils.Wrapped is a Wrapped version of Fusion that adds events that are compatible with all devices. Think a single ``OnPressDown`` event that works with xbox, pc, and mobile with one user-facing event connection.

Reply:
Cool!